stellarphot/gui_tools/comparison_functions.py

```diff
@@ -481,8 +483,8 @@ def save(self):
     def _viewer(self):
         header = ipw.HTML(
             value="""
-        <h2>Click and drag or use arrow keys to pan, use +/- keys to zoom</h2>
-        <h3>Shift-left click (or Crtl-left click)to exclude star as target
+        <h3>Click and drag or use arrow keys to pan, use +/- keys to zoom</h3>
```

Comment:
Should end sentence with period.
```suggestion
        <h3>Click and drag or use arrow keys to pan, use +/- keys to zoom.</h3>
```